tests/apollo/test_skvbc_network_partitioning.py

```diff
@@ -38,7 +38,7 @@ def start_replica_cmd(builddir, replica_id):
             "-i", str(replica_id),
             "-s", statusTimerMilli,
             "-v", viewChangeTimeoutMilli,
-            "-f", '0' # disable time service until intermittent failures are fixed for this test
+            "-f", '1'
```

Comment:
Should we do this after https://github.com/vmware/concord-bft/pull/2102 ?

Reply:
I have not observed any failure due to TS in this test locally after running it multiple times. I'm testing it in CI with enabled TS. Will give all tests multiple CI runs. If any failures are observed due to TS, I will disable TS for it.